src/staging/pg_conversion.py

```diff
@@ -54,14 +55,14 @@ def pg_to_pg_mapper(
     filtered_df[target_col] = filtered_df[pg_column].map(map_dict)
```

Comment:
can we also remove the "to_numeric" now that we're doing validation that does the job? the "coerce" is not very nice cos it puts nans where it can't convert, and we don't want nans in a mapper.

Reply:
This has been removed